src/curate_gpt/cli.py

```diff
@@ -1595,6 +1601,66 @@ def _text_lookup(obj: Dict):
     db.update_collection_metadata(collection, object_type="OntologyClass")
 
 
+@ontology.command(name="subsumption")
```

Comment:
We should decide on one place or another. I think in the agent itself is best since this is more of a custom analysis rather than a generic command?

Reply:
I'd be in favor of having this command available for easy results replication _if_ lines 1647 onward go in the agent code

Reply:
+1 thanks @caufieldjh!